docs/checks/RoadNameGapCheck.md

```diff
@@ -0,0 +1,37 @@
+# Road name gap check
```

Comment:
We also have a document https://github.com/osmlab/atlas-checks/blob/dev/docs/available_checks.md with all the available checks. Could you add add this check to the available_checks.md doc as well?

Reply:
Done.